index.d.ts

```diff
@@ -1,32 +1,37 @@
-interface BaseSignOptions {
-  app: string;
-  identity?: string;
-  platform?: string;
-  keychain?: string;
-}
+declare module "electron-osx-sign" {
+  interface BaseSignOptions {
+    app: string;
+    identity?: string;
+    platform?: string;
+    keychain?: string;
+  }
 
-interface SignOptions extends BaseSignOptions {
-  binaries?: string[];
-  entitlements?: string;
-  'entitlements-inherit'?: string;
-  'gatekeeper-assess'?: boolean;
-  ignore?: string;
-  'pre-auto-entitlements'?: boolean;
-  'pre-embed-provisioning-profile'?: boolean;
-  'provisioning-profile'?: string;
-  'requirements'?: string;
-  'type'?: string;
-  version?: string;
-}
+  interface SignOptions extends BaseSignOptions {
+    binaries?: string[];
+    entitlements?: string;
+    'entitlements-inherit'?: string;
+    'gatekeeper-assess'?: boolean;
+    ignore?: string;
+    'pre-auto-entitlements'?: boolean;
+    'pre-embed-provisioning-profile'?: boolean;
+    'provisioning-profile'?: string;
+    'requirements'?: string;
+    'type'?: string;
+    version?: string;
+    skipIdentityValidation?: boolean;
```

Comment:
@develar I am not sure if to order the options alphabetically or just by priority/usage frequency with `skipIdentityValidation`.